libft/ft_strchr.c

```diff
@@ -6,12 +6,28 @@
 /*   By: dsaripap <marvin@codam.nl>                   +#+                     */
 /*                                                   +#+                      */
 /*   Created: 2019/03/30 14:23:52 by dsaripap      #+#    #+#                 */
-/*   Updated: 2019/05/02 13:27:49 by dsaripap      ########   odam.nl         */
+/*   Updated: 2020/08/04 19:30:13 by svan-der      ########   odam.nl         */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "../includes/libft.h"
 
+size_t	ft_strchri(const char *str, char c)
```

Comment:
If this is another function it should not be put in the same file as ft_strchr

Reply:
Oke, my own set up in my libft is in that way. I can make another file for ft_strchri.